pkg/domain/infra/abi/images.go

```diff
@@ -92,15 +92,17 @@ func (ir *ImageEngine) Prune(ctx context.Context, opts entities.ImagePruneOption
 }
 
 func toDomainHistoryLayer(layer *libimage.ImageHistory) entities.ImageHistoryLayer {
-	l := entities.ImageHistoryLayer{}
-	l.ID = layer.ID
+	l := entities.ImageHistoryLayer{
+		Comment:   layer.Comment,
+		CreatedBy: layer.CreatedBy,
+		ID:        layer.ID,
+		Size:      layer.Size,
+		Tags:      make([]string, len(layer.Tags)),
```

Comment:
is there a reason to copy the slice? why not just `Tags: layer.Tags`

Reply:
I didn't check the git history but assume we could safely use `layers.Tags`.